index.html

```diff
@@ -10,50 +10,17 @@
 <body>
     <main>
         
-        <header>  
-
-            <a href="#" class="logo">Welcome</a> 
-            <div class="container"> 
-                    
+        <header>
+            
+            <a href="#" class="logo">Welcome</a>
+     
             <nav class="menu-bar">
-               
-             <div class="nav-heading">             
-                <div class="nav-bar">
-                    <div class="bar line-1"></div>
-                    <div class="bar line-2"></div>
-                    <div class="bar line-3"></div>
-                </div>
-              </div>
-
-              <ul class="nav-list">
-                <li class="nav-item">
-                    <a class="nav-link" href="#portfolio">
-                        Portfolio
-                    </a>
-                </li>
-
-                <li class="nav-item">
-                    <a class="nav-link" href="#about-me">
-                        About
-                    </a>
-                </li>
-
-                <li class="nav-item">
-                    <a class="nav-link" href="#contact-me">
-                        Contact
-                    </a>
-                </li>
-            </ul>
-
-             </nav>
-
-            </div>
+                <img class="bar" src="asset/images/Icon-Menu.png" alt="menu Icon">
```

Comment:
- [ ] Please kindly fix your hamburger menu, it has to work as follows:

- When the user clicks (or taps) the hamburger button, the mobile menu appears

- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.
![harm](https://user-images.githubusercontent.com/55596970/167675386-4f96ef28-eb9b-40fe-b643-955981ea8b14.png)

Reply:
@CollinsTatang ,

I just checked and the mobile menu hamburger is working well on my computer. I don't know what could be the problem. Please,  can you re-check?

Reply:
Hi @udeaghad 
I just check the branch with the pull request the hamburger menu is not working.